packages/tpetra/core/test/Tpetra_TestingUtilities.hpp

```diff
@@ -117,15 +117,18 @@ namespace Tpetra {
       }
     }
 
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
     /// \brief Get the default Kokkos(Classic) Node for Tpetra tests.
     /// \tparam Node The Kokkos(Classic) Node type.
     ///
     /// \warning This function is an implementation detail of Tpetra.
     ///   Users must not call this function or rely on its behavior.
     template <class Node>
+    TPETRA_DEPRECATED
     Teuchos::RCP<Node> getNode () {
```

Comment:
This function is an implementation detail of Tpetra's tests, so I don't think we need to deprecate it (though it doesn't hurt).  See also the notes above about calling deprecated methods (like Node's constructor, which should be deprecated if it's not already).

Reply:
Now returns Teuchos::null
I'll go with "though it doesn't hurt" for the rest.